src/test/java/com/wikia/webdriver/testcases/facebooktests/FacebookTests.java

```diff
@@ -51,77 +35,10 @@ public void userCanSignUpViaFacebook() {
     String password = "Pass" + signUp.getTimeStamp();
     String email = credentials.emailQaart2;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "email" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1481)